.github/workflows/cypress-integration.yml

```diff
@@ -0,0 +1,47 @@
+name: 'Cypress Integration Tests'
+
+on: [push]
+
+jobs:
+  cypressIntegration:
+    services:
+      mongo:
+        image: mongo
+        ports:
+          - 27017:27017
+
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout each repo into sub-directories
+      - uses: actions/checkout@v2
+        with:
+          repository: conveyal/analysis-ui
+          ref: dev
+          path: ui
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          path: r5
+
+      # Build .jar and copy to ./ui directory
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - run: gradle shadowJar -x test
+        working-directory: r5
+      - run: cp $(ls ./r5/build/libs/*-all.jar | head -n1) ./ui/latest.jar
+
+      # Install / cache dependencies with Cypress to handle caching Cypress binary.
```

Comment:
Which line below does this comment refer to? I'm trying to determine which Actions have caching behavior, to see if it's affecting Cypress test duration. https://github.com/conveyal/analysis-ui/blob/dev/.github/workflows/lint-jest-build.yml has a few `actions/cache@v2` steps with paths like `${{ github.workspace }}/.next`. Would something like that be beneficial here?

Reply:
I could make that comment a bit more verbose. The Cypress Github Action handles the dependency installation and caching, which isn't that hard to set up for a normal `node_modules` cache, but with the Cypress binary is a bit trickier.

Reply:
It could be helpful to update it, just to better understand what's being cached (not just Cypress itself and its dependencies, but also dependencies for the rest of the project?) I wasn't quite sure why there were separate cache steps in the analysis-ui workflow if the Cypress action takes care of caching.